src/Turing.jl

```diff
@@ -61,8 +61,9 @@ include("deprecated.jl") # to be removed in the next minor version release
 # Exports #
 ###########
 # `using` statements for stuff to re-export
-using DynamicPPL: pointwise_loglikelihoods, generated_quantities, logprior, logjoint
+using DynamicPPL: pointwise_loglikelihoods, generated_quantities, logprior, logjoint, condition, decondition, fix, unfix, OrderedDict, conditioned
```

Comment:
It's a bit dangerous to re-export indirect dependencies. I think it would be better to use
```suggestion
using DynamicPPL: pointwise_loglikelihoods, generated_quantities, logprior, logjoint, condition, decondition, fix, unfix, conditioned
using OrderedCollections: OrderedDict
```

Reply:
Good point; done!